### PR TITLE
XRDDEV-1932-2091 Merge conflict fixes

### PR DESCRIPTION
**Pull request by vellotis**
`developt-7.x` **==(PULL)=>** `XRDDEV-1932-2091-add-members-and-identifiers-refacto` **=** `XRDDEV-1932-2091-add-members-and-identifiers-refacto-merge-conflict-fix`

### CODE REVIEW

File: ansible/roles/compile-module/tasks/main.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 - name: Compile xroad modules {{selected_modules}}
-  command: ../src/compile_module.sh {{modules[item].dir}} {{modules[item].build}}
+  command: ../src/compile_module.sh {{modules[item].dir}} {{modules[item]build}}
```
Comment (mloitm) [inline ansible/roles/compile-module/tasks/main.yml:3]: Did yo mean to change this? Looks like accidental find/replace byproduct.

Comment (andresrosenthal) [reply ansible/roles/compile-module/tasks/main.yml:3]: You are correct, thanks for the find!

File: src/systemtest/op-monitoring/xrd-mock-soapui/mock-client-pom.xml [hunk 2]
```diff
@@ -1,77 +1,77 @@
...
+                        <value>ClientSubsystemCode=${ClientSubsystemCode}</value>
+                        <value>ClientSubsystemCodeTLS=${ClientSubsystemCodeTLS}</value>
+                        <value>ServiceXRoadInstance=${ServiceXRoadInstance}</value>
+                        <value>ServiceMemberClass=${ServiceMemberClass}</value>
+                        <value>ServiceMemberCode=${ServiceMemberCode}</value>
+                        <value>ServiceSubsystemCode=${ServiceSubsystemCode}</value>
+                        <value>ServiceSubsystemCodeTLS=${ServiceSubsystemCodeTLS}</value>
+                    </projectProperties>
+                    <printReport>true</printReport>
+                    <outputFolder>${projectbuild.directory}/surefire-reports</outputFolder>
```
Comment (mloitm) [inline src/systemtest/op-monitoring/xrd-mock-soapui/mock-client-pom.xml:69]:
```suggestion
                    <outputFolder>${project.build.directory}/surefire-reports</outputFolder>
```

Comment (andresrosenthal) [reply src/systemtest/op-monitoring/xrd-mock-soapui/mock-client-pom.xml:69]: 👍

File: src/centralserver/admin-ui/license-check-and-add-config.json [hunk 3]
```diff
@@ -23,7 +23,7 @@
     "scripts/dtsgen-wrapper.js",
     "tests/e2e/testdata/",
     "build/",
-    ".build/"
+    "build/"
```
Comment (mloitm) [inline src/centralserver/admin-ui/license-check-and-add-config.json:26]: duplicate entry

**Pull request merged**